man/man8/zfs.8

```diff
@@ -785,12 +785,11 @@ don't wait.
 .Xr fsync 2 ,
 .Xr stat 2 ,
 .Xr write 2 ,
-.Xr acl 5 ,
-.Xr attributes 5 ,
+.Xr acl 3 ,
```

Comment:
It seem these references might vary across platforms.  At least for Linux here's what I would expect.

https://man7.org/linux/man-pages/man5/acl.5.html
https://man7.org/linux/man-pages/man7/attributes.7.html

Reply:
Yes, agree. 

The acl(5) [Linux] acl(3) [FreeBSD]
It does look like in both instances the attributes 5 is incorrect, as per your above link.

What would be the recommended next steps to address this discrepency? I think the deltas in the section numbers will be an on-going theme.

Reply:
Agreed, probably the best option would be to do [something like this](https://github.com/openzfs/zfs/pull/15017) so the man page is generated with the right content for the platform its built on.

Reply:
after reading that recommendation, i would have to make this pull request use the FREEBSD macro to allow the differences, is that correct?

Reply:
Right.  We'd also have to dust off and merge #15017 first.